srcpkgs/apparmor-rules-void/template

```diff
@@ -0,0 +1,20 @@
+# Template file for 'apparmor-rules-void'
+pkgname=apparmor-rules-void
+version=2021.05.17
+revision=1
+build_style=meta
+conf_files="/etc/apparmor.d/local/*"
+short_desc="AppArmor Void Linux rules"
+maintainer="Paper <paper@tilde.institute>"
+license="GPL-2.0-only"
+homepage="https://github.com/void-linux/void-packages/"
+
+do_install() {
+	vmkdir etc/apparmor.d/local
+	cp ${FILESDIR}/profiles/* ${DESTDIR}/etc/apparmor.d/
+	cd ${DESTDIR}/etc/apparmor.d/
+	find . -maxdepth 1 -type f | while read -r rulepath; do
+		rule="${rulepath/.\/}"
+		echo "# Site-specific additions and overrides for '$rule'" > "local/$rule"
```

Comment:
I don't think we should ship local files at all, IMHO better to just `include if exist`.
(They also need to be marked as configuration file in case we keep shipping them).

Reply:
I wanted to match the behavior of the upstream rules, do you think I should remove it from there as well?